scripts/cti

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+aws codebuild list-projects >/dev/null || (echo "Failed to access codebuild, try awsmfa?"; exit 1)
```

Comment:
These checks can be moved to after flag parsing checks so that these don't run when incorrect flags are input by the user

Reply:
I actually feel like if user environment is incorrect its better to give feedback earlier, since it takes more time to fix environment, then to figure out flags for script (also note that we only check pretty much one flag (`--pr`), everything else is not checked in this script and ultimately only verified in cluster test vinary